torchgen/gen.py

```diff
@@ -2076,12 +2086,38 @@ def gen_backend_select() -> Dict[str, List[str]]:
     schema_selector = selector
     if force_schema_registration:
         schema_selector = SelectiveBuilder.get_nop_selector()
+    schema_static_template = CodeTemplate(
+        """
+TORCH_LIBRARY($ns, m) {
+    $schema_registrations_body
+};"""
+    )
+    ns_native_functions: Dict[str, List[NativeFunction]] = defaultdict(list)
+    for native_function in native_functions:
+        ns_native_functions[native_function.namespace].append(native_function)
+    schema_registrations = ""
+    aten_schema_registrations = []
+    for namespace, functions in ns_native_functions.items():
+
+        schema_registrations_body = list(
+            mapMaybe(RegisterSchema(schema_selector), functions)
+        )
+        if namespace == "aten":
+            aten_schema_registrations = schema_registrations_body
+        else:
+            schema_registrations += schema_static_template.substitute(
```

Comment:
It looks like this gives the yaml author total freedom to add multiple different namespaces if they want to. Do we actually need that today? If we do then I think this is fine, but if you only really need "aten + one other custom namespace" for mobile today, then I think we should be keep it simpler and limit the codegen to that.